flepimop/gempyor_pkg/src/gempyor/outcomes.py

```diff
@@ -151,111 +151,117 @@ def read_parameters_from_config(modinf: model_info.ModelInfo):
                         f"Places in seir input files does not correspond to subpops in outcome probability file {branching_file}"
                     )
 
-        subclasses = [""]
-        if modinf.outcomes_config["subclasses"].exists():
-            subclasses = modinf.outcomes_config["subclasses"].get()
+        # subclasses = [""]
+        # if modinf.outcomes_config["subclasses"].exists():
+        #    subclasses = modinf.outcomes_config["subclasses"].get()
 
         parameters = {}
         for new_comp in outcomes_config:
             if outcomes_config[new_comp]["source"].exists():
-                for subclass in subclasses:
-                    class_name = new_comp + subclass
-                    parameters[class_name] = {}
-                    # Read the config for this compartement
-                    src_name = outcomes_config[new_comp]["source"].get()
-                    if isinstance(src_name, str):
-                        if src_name != "incidI":
-                            parameters[class_name]["source"] = src_name + subclass
-                        else:
-                            parameters[class_name]["source"] = src_name
+                # for subclass in subclasses:
+                #    class_name = new_comp + subclass
+                #    parameters[class_name] = {}
+                parameters[new_comp] = {}
+                # Read the config for this compartement
+                src_name = outcomes_config[new_comp]["source"].get()
+                if isinstance(src_name, str):
+                    #    if src_name != "incidI":
+                    #        parameters[class_name]["source"] = src_name + subclass
+                    #        else:
+                    #            parameters[class_name]["source"] = src_name
+                    parameters[new_comp]["source"] = src_name
+                else:
```

Comment:
this should be an `elif` to make sure we handle all case to the value error

Reply:
do you mean it should be changed as:

  if isinstance(src_name, str):
  elif ("incidence" in src_name.keys()) or ("prevalence" in src_name.keys()):
  else:  raise ValueError()

?